### ISSUE TITLE
Why is Unity/Assets/AzureSpatialAnchors.SDK dir missing scripts? 2.5.0+

### ISSUE THREAD

**Issue by ghostravenstorm** [Azure/azure-spatial-anchors-samples#239]
The minimum SDK for ASA caught me by surprise. I just started working with the HoloLens and Unity ASA 4 weeks ago and I was following the MRTK tutorials for the HoloLens 2 ([tutorial](https://docs.microsoft.com/en-us/windows/mixed-reality/develop/unity/tutorials/mr-learning-sharing-05)). I downloaded MRTK 2.4.0 and the ASA SDK 2.4.0 following the tutorial then (didn't really see a notice that the minimum version was going to change). I began working on other aspects with my project having ASA working as intended, and very recently I'm surprised that ASA isn't working at all anymore. Not even the MRTK demos I downloaded were working.

So now I'm trying to upgrade my ASA SDK to at least 2.7.0, but that hasn't been easy or straight forward. All the resources just point to this sample repo to download. I remember downloading a Unity package from the MRTK tutorials, but I can't find that link anymore. So I cloned the repo 2.7.2, but I don't understand why the SDK dir is missing scripts. I don't understand what changed between 2.4.0 and 2.7.0. How am I suppose to import this into my projects with no scripts? Are they all in Unity/Assets/AzureSpatialAnchors.Examples now? Those scripts don't seem to match what 2.4.0 had.

**Comment by msftradford** [comment]
Hello @ghostravenstorm, happy to help out with this. The SDK is released separately from the sample code. To configure the ASA SDK in your Unity project, please see [these instructions](https://docs.microsoft.com/en-us/azure/spatial-anchors/how-tos/setup-unity-project). After importing the ASA SDK Unity packages, you should see the scripts in your Packages folder in the Unity Project pane.

**Comment by ghostravenstorm** [comment]
Thanks!
I followed these instructions, but it's showing that refs to `CloudSpatialAnchor` are missing.

- I opened a blank Unity 2019.4.19f1 project
- Switched the build to UWP HoloLens, ARM64
- Imported AR Foundation and Windows Mixed Reality
- Then imported the Azure Spatial Anchors SDK Core 2.7.2

Now I see compile errors where `CloudSpatialAnchor` is mssing.

![image](https://user-images.githubusercontent.com/14009559/107558483-d52b6200-6ba8-11eb-9e67-af8eb691c0da.png)
